.github/workflows/auto_update_version.yml

```diff
@@ -0,0 +1,57 @@
+name: Update PrintVersion on release branch creation
+
+on:
+  create:
+    branches:
+      - "release/*"
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract version from branch name
+        run: |
+          BRANCH_NAME="${GITHUB_REF_NAME}"
+          VERSION="${BRANCH_NAME#release/}"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "RELEASE_BRANCH=$BRANCH_NAME" >> $GITHUB_ENV
+          echo "UPDATE_BRANCH=updateversion/$VERSION" >> $GITHUB_ENV
+
+      - name: Create updateversion branch
+        run: |
+          git checkout -b $UPDATE_BRANCH $RELEASE_BRANCH
+
+      - name: Update PrintVersion.swift
+        run: |
+          # Update the version string in PrintVersion.swift
+          sed -i "s/print(\".*\")/print(\"$VERSION\")/" Sources/swift-format/PrintVersion.swift
+
+      - name: Commit and push changes
+        run: |
+          git config user.name "swift-ci"
+          git config user.email "swift-ci@users.noreply.github.com"
+          git add Sources/swift-format/PrintVersion.swift
+          git commit -m "Update version to $VERSION"
+          git push origin $UPDATE_BRANCH
+
+      - name: Create Pull Request
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh pr create \
+            --base "${{ env.RELEASE_BRANCH }}" \
+            --head "${{ env.UPDATE_BRANCH }}" \
+            --title "[${{ env.VERSION }}] Update version to ${{ env.VERSION }}" \
+            --body "This PR updates the version in PrintVersion.swift to ${{ env.VERSION }}.
+
+          Automatically created when release branch ${{ env.RELEASE_BRANCH }} was created." \
```

Comment:
How about the following:
> This PR was automatically opened by a GitHub action on creation of a release branch (${{ env.RELEASE_BRANCH }}). Review the version update and merge if correct, otherwise update the version manually.

?

Reply:
Yes, I’ve updated it. Thank you!